config/api_platform/resources.yaml

```
@@ -1,4 +1,64 @@
 resources:
+  App\Customer\Domain\Entity\CustomerStatus:
+    paginationPartial: true,
+    paginationClientItemsPerPage: true
+    paginationViaCursor:
+      - { field: 'ulid', direction: 'DESC' }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)